src/azure-cli/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -436,30 +419,35 @@ def cli_migration_start(client, resource_group_name, namespace_name, target_name
 timedeltapattern = re.compile("^\\d+:\\d+:\\d+$")
 
 
-def return_valid_duration(objinstance, duration_property, update_value):
+def return_valid_duration(instance_value, update_value):
     from datetime import timedelta
     from isodate import parse_duration
     if update_value is not None:
-        if iso8601pattern.match(update_value):
-            for attr, value in vars(objinstance).items():
-                if attr == duration_property:
-                    if parse_duration(update_value) < timedelta(days=10675199, minutes=10085, seconds=477581):
-                        return update_value
+        value_toreturn = update_value
+    else:
+        value_toreturn = str(instance_value)
 
-                    if parse_duration(update_value) > timedelta(days=10675199, minutes=10085, seconds=477581):
-                        return None
+    if iso8601pattern.match(value_toreturn):
+        if parse_duration(value_toreturn) < timedelta(days=10675199, minutes=10085, seconds=477581):
```

Comment:
Can we put this duration in a constant, instead of repeating it?

Reply:
used constant